plugins/woocommerce/client/blocks/tests/e2e/tests/templates/template-customization.block_theme.spec.ts

```diff
@@ -115,9 +121,17 @@ test.describe( 'Template customization', () => {
 				await admin.visitSiteEditor( {
 					postType: testData.templateType,
 				} );
-				await editor.revertTemplate( {
-					templateName: testData.fallbackTemplate?.templateName || '',
-				} );
+				if ( testData.templateType === 'wp_template' ) {
+					await editor.revertTemplate( {
+						templateName:
+							testData.fallbackTemplate?.templateName || '',
+					} );
+				} else {
+					await editor.revertTemplatePart( {
+						templateName:
+							testData.fallbackTemplate?.templateName || '',
+					} );
+				}
```

Comment:
`testData.fallbackTemplate` will always we a `wp_template`, if I'm not missing anything. So this if would be unnecessary:


```suggestion
				await editor.revertTemplate( {
					templateName:
						testData.fallbackTemplate?.templateName || '',
				} );
```